_data/simple-icons.json

```
@@ -2764,8 +2764,8 @@
         },
         {
             "title": "CurseForge",
-            "hex": "6441A4",
-            "source": "https://www.curseforge.com"
+            "hex": "F16436",
+            "source": "https://curseforge.com"
```

```suggestion
            "source": "https://www.curseforge.com"
```

Site still uses www. - so would be best if we kept it in the URL.

---

Done